src/controllers/album.js

```diff
@@ -0,0 +1,63 @@
+const db = require('../db/index');
+
+const createAlbum = async (req, res) => {
+  const { name, date } = req.body;
+  const { id } = req.params;
+
+  try {
+    const {
+      rows: [album],
+    } = await db.query(
+      'INSERT INTO Albums (name, date, artistId) VALUES ($1, $2, $3) RETURNING *',
+      [name, date, id]
+    );
+    res.status(201).json(album);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const getAlbums = async (req, res) => {
```

Comment:
parameter req can be replaced with _ as it's not used in the function, makes the code more readable